src/main/java/com/razorpay/ApiClient.java

```diff
@@ -35,35 +35,35 @@ class ApiClient {
     this.auth = auth;
   }
 
-  public <T extends Entity> T get(String path, JSONObject requestObject) throws RazorpayException {
-    Response response = ApiUtils.getRequest(path, requestObject, auth);
+  public <T extends Entity> T get(String version, String path, JSONObject requestObject) throws RazorpayException {
+    Response response = ApiUtils.getRequest(version, path, requestObject, auth);
     return processResponse(response);
   }
 
-  public <T extends Entity> T post(String path, JSONObject requestObject) throws RazorpayException {
-    Response response = ApiUtils.postRequest(path, requestObject, auth);
+  public <T> T post(String version, String path, JSONObject requestObject) throws RazorpayException {
```

Comment:
Made these changes because this [api](https://razorpay.com/docs/partners/aggregators/partner-auth/token-sharing/#delete-a-token) is returning an empty array `[]`  also checking these method checking the return type . so after these changes now it supports empty `[]`